manager/src/main/java/com/redhat/service/bridge/manager/models/Processor.java

```diff
@@ -168,4 +187,15 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id);
     }
+
+    private Set<com.redhat.service.bridge.infra.models.filters.Filter> buildFilters() {
```

Comment:
I would like to avoid class name clash, can you please rename one of the two `Filter` class?

Reply:
Renamed the infra one to  `BaseFilter`